docs/manual/kinds/ceos.md

```diff
@@ -39,13 +39,24 @@ Arista cEOS node launched with containerlab can be managed via the following int
 !!!info
     Default user credentials: `admin:admin`
 
+## Interface naming and link definition
+
+When defining links in the topology, ceos container interfaces should be named `et` as opposed to `eth`.  This provides consistency with the underlying interface mappings and the interactions with some features internally to ceos.  This naming requirement does not apply to the `eth0` interface created by containerlab only to links that are used for interconnection with other elements in a topology.
+
+example:
+```yml
+  links:
+    - endpoints: ["ceos_rtr1:et1", "ceos_rtr2:et1"]
+    - endpoints: ["ceos_rtr1:et2", "ceos_rtr3:et1"]
+```
+
 ## Interfaces mapping
 ceos container uses the following mapping for its linux interfaces:
 
 * `eth0` - management interface connected to the containerlab management network
-* `eth1` - first data interface
+* `et1` - first data interface
 
-When containerlab launches ceos node, it will set IPv4/6 addresses as assigned by docker to the `eth0` interface and ceos node will boot with that addresses configure. Data interfaces `eth1+` need to be configured with IP addressing manually.
+When containerlab launches ceos node, it will set IPv4/6 addresses as assigned by docker to the `et0` interface and ceos node will boot with that addresses configure. Data interfaces `et1+` need to be configured with IP addressing manually.
```

Comment:
since `eth0` interface is not part of this PR, it shouldn't have changed here in this sentence I suppose?

Reply:
ah! good catch.  i'll revise this.

Reply:
updated text accordingly.